src/App.tsx

```diff
@@ -3,12 +3,35 @@ import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
-type State = {};
+type State = {
+  query: string;
+};
 
 export class App extends React.Component<{}, State> {
-  state: State = {};
+  state: State = {
+    query: '',
+  };
+
+  searchHandler = (event: React.ChangeEvent<HTMLInputElement>) => {
+    this.setState({
+      query: event.currentTarget.value,
+    });
+  };
+
+  sortHandler = () => {
```

Comment:
you filter movies, you don't sort them

```suggestion
  getFilteredMovies = () => {
```